systers_portal/templates/account/logout.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+{% load url from future %}
```

Comment:
This syntax is deprecated and will be removed in Django 1.9.
[Proof link](https://docs.djangoproject.com/en/1.7/releases/1.7/#loading-ssi-and-url-template-tags-from-future-library).